src/olympia/conf/dev/settings.py

```diff
@@ -64,6 +64,24 @@
 SYSLOG_TAG2 = "http_app_addons_dev_timer"
 SYSLOG_CSP = "http_app_addons_dev_csp"
 
+NETAPP_STORAGE_ROOT = env('NETAPP_STORAGE_ROOT')
```

Comment:
Not that it matters, but why the settings files reording?

Reply:
Needed `FILESYSTEM_CACHE_ROOT` before `CACHES`, which in turn meant `NETAPP_STORAGE_ROOT` before that, and I figured I'd keep all the storage settings together.